src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -79,9 +79,9 @@ public function load(array $configs, ContainerBuilder $container)
         $config = $this->processConfiguration($configuration, $configs);
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
 
-        $container->setParameter('sulu_core.locales', array_keys($config['locales']));
+        $container->setParameter('sulu_core.locales', array_unique(array_keys($config['locales'])));
         $container->setParameter('sulu_core.translated_locales', $config['locales']);
-        $container->setParameter('sulu_core.translations', $config['translations']);
+        $container->setParameter('sulu_core.translations', array_unique($config['translations']));
```

Comment:
But if these values have the same key, the yml parser should already handle this, shouldn't it?